Improve inputDate.js to allow the management of selectable days

Daynos opened the pull request:
Allow the management of selectable days through function "isSelectable" in dateinput parameters.

Example, to prevent selection of sundays  :
$(".input-date").dateinput({ 
    isSelectable: function (date) {
        if(date.getDay() == 0) return false;
        return true;
    }
});

Selection of days can be improved with more complex functions.

Daynos committed:
Improve inputDate.js 

Allow the management of selectable days through function "isSelectable" in parameters.

Eg : to prevent selection of sundays  :
$(".input-date").dateinput({ 
isSelectable: function (date) {
if(date.getDay() == 0) return false;
return true;
}
});

Selection of days can be improved with more complex functions.

alibby251 commented:
Thanks for this Daynos - I assume at present that this can only apply for single values, i.e. 1 = Monday, and not multiple values such as 1,3,5, and so on?

Daynos committed:
Update scrollable.js

Copy / paste from https://github.com/TylerRick/jquerytools/commit/7b11deb0213a7901b5b558a747a8da872b6286d6 (sorry I don't know how to keep a link to TylerRick's work :/)

Daynos committed:
Update scrollable.js

Daynos commented:
Oups! Didin't see there was a question... Sorry to be so late :/

Any function using 'date' as parameter and returning 'true' or 'false' can be used. To answer your question, you can replace the 'if' with a 'switch':

``` javascript
$(".input-date").dateinput({
    isSelectable: function (date) {
        switch(date.getDay()){
            case 1: 
            case 3: 
            case 5: 
                return false;
            default: 
                return true;
        }
    }
});
```
